### DISCUSSION TITLE
Pool function returns error after updating to R 4.0.3

### DISCUSSION THREAD

**Discussion by BeRouven** [amices/mice#344] [Impute, analyse and pool]
Hi there,

I have used MICE to perform imputation of longitudinal data. To analyse the data I performed a mixed model using the lme function from the nlme package and pooled the results afterwards. This worked well so far, but after updating from R version 3.6.3 to 4.0.3 the pool function returns an error message.

Here I reproduced the issue for the brandsma dataset, which was used to illustrate 2 level imputation:
```
library(mice)
library(miceadds)
library(nlme)

data("brandsma", package = "mice")

d <- brandsma[, c("sch", "lpo", "iqv")]
pred <- make.predictorMatrix(d)
pred["lpo", ] <- c(-2, 0, 1)
pred["iqv", ] <- c(-2, 1, 0)
imp <- mice(d, pred = pred, meth = "2l.pmm", seed = 919,
            m = 10, print = FALSE)

fit <- with(imp, lme(lpo ~ iqv, random = ~1 | sch))
pool(fit)
```

> Error: No tidy method for objects of class lme

Any idea what causes this issue and how to fix it?
Thanks in advance!

**Comment by stefvanbuuren** [comment]
See #274 and related.
